ConsoleApp1/Poll.cs

```diff
@@ -0,0 +1,104 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace ConsoleApp1
+{
+    internal class Poll
+    {
+        public string PollName { get; private set; }
+        public string PollQuestion { get; private set; }
+        public List<Variant> variants { get; private set; } = new List<Variant>();
+        public int VotesYay { get; private set; }
+        public int VotesNay { get; private set; }
+        public enum MyEnum
+        {
+            Single,
+            Complex
+        }
+        public MyEnum my;
+        public Poll()
```

Comment:
bad decision to place logic into the constructor, either create builder either create fabric method